client/Components/navbar.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Navbar = props => {
+
+  return (
+    <div className="row navbar-map">
+      <i className="fas fa-users fa-3x"></i>
+      <i className="fas fa-globe fa-3x"></i>
+      <i className="fas fa-user fa-3x"></i>
+    </div>
+  );
+};
+
+export default Navbar;
```

Comment:
Please leave this out until it does something. It's really important not to include code that isn't actually ready for review.